src/platforms/flutter/common/configuration/integrations/sqflite-instrumentation.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: sqflite Database Instrumentation
+caseStyle: camelCase
+supportLevel: production
+sdk: sentry.dart.sqflite
+description: "Learn more about the Sentry sqflite Database Instrumentation for the Flutter SDK."
+categories:
+  - mobile
+---
+
+<Include name="beta-note.mdx" />
```

Comment:
`m`: do we actually beta this? if so, do we usually use `supportLevel: production` for beta integrations?

Reply:
`supportLevel` is only `production` or `community, and not really related to the support level similar to semver.

Reply:
I'd keep it beta at least for a few days/weeks since we use some internal APIs, we need to be sure that we are not breaking any behavior for any platform.